packages/beacon-node/src/api/impl/debug/index.ts

```diff
@@ -85,5 +86,21 @@ export function getDebugApi({
         },
       };
     },
+    async getHistoricalSummaries(_, context) {
+      const {state} = await getStateResponseWithRegen(chain, "head");
```

Comment:
I don't think hard coding this to `head` is not great, we should probably align this with other apis which take a `state_id` parameter

Reply:
correct, stateid in the route and corresponding change here makes sense

Reply:
Will do.  I also need to add an additional object to the response, a single proof that anchors the `historicalSummaries` property in the `BeaconState` and that will take me a bit to correctly figure out.  I have the code for the proof already setup on the ultralight side but need to match it up here.

Reply:
Have updated here as well, adding the proof that anchors the `HistoricalSummaries` object in the `BeaconState`.